services/generic/alarm.md

```diff
@@ -1,41 +1,80 @@
+| `alarm_lock`        | manual_lock, manual_unlock, rf_lock, rf_unlock, keypad_lock, keypad_unlock, tag_lock, tag_unlock, manual_not_locked, rf_not_locked, auto_locked, jammed, door_opened, door_closed, lock_failed                                                                                           |
+| `alarm_power`       | on, ac_on, ac_off, surge, voltage_drop, over_current, over_voltage, replace_soon, replace_now, charging, charged, charge_soon, charge_now                                                                                                                                                |
+| `alarm_siren`       | inactive, siren_active                                                                                                                                                                                                                                                                   |
+| `alarm_system`      | hw_failure, sw_failure, hw_failure_with_code, sw_failure_with_code                                                                                                                                                                                                                       |
+| `alarm_time`        | wakeup, timer_ended, time_remaining                                                                                                                                                                                                                                                      |
+| `alarm_water_valve` | valve_op, master_valve_op, valve_short_circuit, current_alarm, alarm_water_valve, master_valve_current_alarm                                                                                                                                                                             |
+| `alarm_water`       | leak, level_drop, replace_filter                                                                                                                                                                                                                                                         |
+| `alarm_weather`     | inactive, moisture                                                                                                                                                                                                                                                                       |
+
+## Interfaces
+
+| Type | Interface            | Value type | Storage | Description                                                                                                 |
+|------|----------------------|------------|---------|-------------------------------------------------------------------------------------------------------------|
+| in   | cmd.alarm.get_report | str_map    |         | Requests [`event`](#service-names) status. See [`event_request`](#definitions) definition for more details. |
```

Comment:
It might be taken under consideration to point out that Z-Wave devices in most cases will not respond to cmd.alarm.get_report, or response will be undefined.
In Z-Wave, devices only reports alarms/notifications. (There is a small exception, but even Silicon Labs Support could not point me any device that would support getting alarm reports).